src/main/java/com/lothrazar/storagenetwork/block/main/NetworkModule.java

```diff
@@ -77,16 +77,18 @@ public Set<IConnectable> getConnectables() {
     return result;
   }
 
+  public List<ItemStack> getStacks() {
+    return getStacks(true);
+  }
   /**
    * returns huge unsorted list of stacks
    * 
    * respects local node filter settings
    * 
    * @return
    */
-  public List<ItemStack> getStacks() {
+  public List<ItemStack> getStacks(boolean isFiltered) {
```

Comment:
yes good call, i should have added this as an arg originally instead of hardcoding it